### ISSUE TITLE
Training fails instantly with KeyError: 'x'

### ISSUE THREAD

**Issue by justinthec** [cloud-annotations/training#166]
Hi, I'm training an Object detection model by following the tutorial. I did one successful training run with images that had a labeled bounding box. However, when I tried a second training run with images that I labeled as "Mark as Negative" with 0 bounding boxes on it, the training failed instantly with this error:

```
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/opt/conda/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/job/model-code/src/data/prepare_data_object_detection.py", line 200, in <module>
  File "/job/model-code/src/data/prepare_data_object_detection.py", line 148, in main
  File "/job/model-code/src/data/prepare_data_object_detection.py", line 112, in create_tf_record
KeyError: 'x'
69 training and 30 validation examples.
CACLI-TRAINING-FAILED
Training exited with error code 1
Failed: learner_exit_code: 1
```

I suspect that the code in `prepare_data_object_detection.py` is assuming that I have a bounding box for each image and is trying to access the x coordinate of a `None` bounding box.

**EDIT**: I've taken a look at the file in question and it looks like [a fix was made in October](https://github.com/cloud-annotations/training/commit/dd21a8214f1f135020e6859f96279398ee556829#diff-be5516fbc2fbad89d149911cb954885b) but it hasn't been deployed to the IBM machines that I'm training on?

**Comment by bourdakos1** [comment]
Thanks, the “mark as negative” feature was released together with the updated code you’ve linked.

I already have an issue open https://github.com/cloud-annotations/training/issues/145

But these logs are helpful!

Can you verify that you have at least cacli version 1.1.10?

**Comment by bourdakos1** [comment]
the training zip didn't get packaged with the new code 🤦‍♂️

thanks for the help!

**Comment by bourdakos1** [comment]
should be fixed in cacli v1.2.1

please reopen if you get the same issue

**Comment by justinthec** [comment]
My pleasure Nick, thanks for the quick response!
